src/pages/SwapNew/components/SwapInfo/Indicator.tsx

```diff
@@ -0,0 +1,58 @@
+import styled from 'styled-components'
+
+import { ReactComponent as BananaSVG } from '../../../../assets/swapbox/banana.svg'
+import { ReactComponent as DexesSVG } from '../../../../assets/swapbox/dexes.svg'
+import { ReactComponent as GasSVG } from '../../../../assets/swapbox/gas.svg'
+import { ReactComponent as MagnifyingGlassSVG } from '../../../../assets/swapbox/magnifying-glass.svg'
+import { ReactComponent as ShieldSVG } from '../../../../assets/swapbox/shield.svg'
+import {
+  INDICATOR_BACKGROUND_COLOR,
+  INDICATOR_COLOR,
+  IndicatorColorVariant,
+  IndicatorIconVariant,
+  FontFamily,
+} from '../../constants'
+
+export const INDICATOR_ICON = {
+  [IndicatorIconVariant.DEXES]: <DexesSVG />,
+  [IndicatorIconVariant.GAS]: <GasSVG />,
+  [IndicatorIconVariant.BANANA]: <BananaSVG />,
+  [IndicatorIconVariant.SHIELD]: <ShieldSVG />,
+  [IndicatorIconVariant.MAGNIFYING_GLASS]: <MagnifyingGlassSVG />,
+}
```

Comment:
Optional: Better to make it all these constants readonly

```suggestion
export const INDICATOR_ICON = {
  [IndicatorIconVariant.DEXES]: <DexesSVG />,
  [IndicatorIconVariant.GAS]: <GasSVG />,
  [IndicatorIconVariant.BANANA]: <BananaSVG />,
  [IndicatorIconVariant.SHIELD]: <ShieldSVG />,
  [IndicatorIconVariant.MAGNIFYING_GLASS]: <MagnifyingGlassSVG />,
} as const
```

Reply:
I agree, good suggestion! 👍🏼

Reply:
Done.